samba.yaml

```
@@ -0,0 +1,417 @@
+package:
+  name: samba
+  version: 4.19.4
+  epoch: 0
+  description: "Tools to access a server's filespace and printers via SMB"
+  copyright:
+    - license: GPL-3.0-or-later AND LGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - acl-dev
+      - autoconf
+      - automake
+      - bind-dev
+      - bison
+      - build-base
+      - busybox
+      - cups-dev
+      - dbus-dev
+      - docbook-xml
+      - e2fsprogs-dev
+      - flex
+      - gnutls-dev
+      - iniparser-dev
+      - jansson-dev
+      - ldb-dev
+      - libarchive-dev
+      - libcap-dev
+      - libtirpc-dev
+      - liburing-dev
+      - linux-pam-dev
+      - ncurses-dev
+      - ncurses-terminfo
+      - ncurses-terminfo-base
+      - openldap-dev
+      - perl
+      - perl-json
+      - perl-parse-yapp
+      - pkgconf-dev
+      - popt-dev
+      - py3-dnspython
+      - py3-ldb
+      - py3-markdown
+      - py3-tdb
+      - py3-tevent
+      - python3-dev
+      - rpcgen
+      - subunit-dev
+      - talloc-dev
+      - tdb
+      - tdb-dev
+      - tevent-dev
+      - zlib-dev
+  environment:
+    IDMAP_MODULES: idmap_ad,idmap_rid,idmap_adex,idmap_hash,idmap_tdb2
+    PDB_MODULES: pdb_tdbsam,pdb_ldap,pdb_ads,pdb_smbpasswd,pdb_wbc_sam,pdb_samba4
+    AUTH_MODULES: auth_unix,auth_wbc,auth_server,auth_netlogond,auth_script,auth_samba4
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 4026d93b866db198c8ca1685b0f5d52793f65c6e63cb364163af661fdff0968c
+      uri: https://download.samba.org/pub/samba/stable/samba-${{package.version}}.tar.gz
+
+  - uses: patch
+    with:
+      patches: tinfo.patch
+
+  - uses: autoconf/configure
+    with:
+      opts: |
```

```suggestion
      opts: |
        LDFLAGS="-ltinfo" \
```

So this is an interesting one - is the `tinfo.patch` above needed if this LDFLAGS approach is used?